support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc

```diff
@@ -30,6 +30,13 @@ include::modules/telemetry-consequences-of-disabling-telemetry.adoc[leveloffset=
 
 include::modules/insights-operator-new-pull-secret-disabled.adoc[leveloffset=+1]
 
+include::modules/insights-operator-register-unconnected-cluster.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../../support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc#telemetry-consequences-of-disabling-telemetry_opting-out-remote-health-reporting[Consequences of disabling remote health reporting]
+* link:https://access.redhat.com/documentation/en-us/subscription_central/2023/html/getting_started_with_the_subscriptions_service/con-how-does-subscriptionwatch-show-data_assembly-viewing-understanding-subscriptionwatch-data-ctxt[How does the subscriptions service show my subscription data?](Getting Started with the Subscription Service)
```

Comment:
SME query: I included this link so a customer has more info about subscription services. The information pertaining to this doc set is broad, so is the link needed?

Reply:
Having the link is good